apps/public/src/app/(general)/bijlage/actions.ts

```diff
@@ -1,14 +1,5 @@
 'use server'
 
-import { cookies } from 'next/headers'
 import { redirect } from 'next/navigation'
 
-export const postAttachmentForm = async () => {
-  // Delete all session cookies
```

Comment:
Why did you delete this? We still want to do this, no?

Reply:
Not here, but on the last page right?

Reply:
Yeah, that makes sense. Can you write a Jira ticket to add this at a later time?